src/pages/forgotPassword/VerifyEmail.tsx

```diff
@@ -0,0 +1,95 @@
+import { FunctionComponent, useState } from "react";
+import { useDispatch } from "react-redux";
+import { Box, useTheme } from "@mui/material";
+import { FormikValues, useFormikContext } from "formik";
+import { Button, GradientTypography, Typography } from "elements";
+import { TextInputField } from "components";
+import { sendVerificationEmail } from "modules/session";
+
+const VerifyEmail: FunctionComponent = () => {
+  const dispatch = useDispatch();
+  const theme = useTheme();
+  const [showResendLink, setShowResendLink] = useState(true);
+  const {
+    values: { email },
+  } = useFormikContext<FormikValues>();
+
+  const handleEmailResend = () => {
+    setShowResendLink(false);
+
+    dispatch(sendVerificationEmail(email));
+
+    setTimeout(() => setShowResendLink(true), 10000);
+  };
+
+  return (
+    <Box
```

Comment:
I've seen other places with `Container` and others with `Box`, is there any subtle difference between the two in the top wrapper layer?